src/Polysemy/Internal/Combinators.hs

```diff
@@ -24,6 +24,7 @@ module Polysemy.Internal.Combinators
 
 import qualified Control.Monad.Trans.State.Lazy as LS
 import qualified Control.Monad.Trans.State.Strict as S
+import qualified Data.Tuple as S (swap)
```

Comment:
The swap defined in this module was my attemp to get around runFixpoint diverging. I don't think it's necessary anymore.

Reply:
It's not, since the fixpoint tests I added still succeed. Still, maybe we want the lazy swap for `runLazyState`?